test/smtcallback.js

```diff
@@ -66,20 +66,27 @@ tape('SMTCheckerCallback', function (t) {
       sources: input
     });
 
-    // Solidity 0.6.9 comes with z3
     var tests;
     if (!semver.gt(solc.semver(), '0.6.8')) {
       tests = [
         { cb: satCallback, expectations: ['Assertion violation happens here'] },
         { cb: unsatCallback, expectations: [] },
         { cb: errorCallback, expectations: ['BMC analysis was not possible'] }
       ];
-    } else {
+    // Solidity 0.6.9 comes with z3
```

Comment:
Shouldn't the comment befurther up?

Reply:
I moved it down here because this is the block that refers to 0.6.9. The block above has the old old behavior, <0.6.9.